src/Connectors/ConnectionFactory.php

```diff
@@ -26,27 +26,19 @@ class ConnectionFactory extends Base
      *
      * @throws \InvalidArgumentException
      */
-    protected function createConnection($driver, $connection, $database, $prefix = '', array $config = [])
-    {
+    protected function createConnection(string $driver, \PDO|\Closure $connection, string $database, string $prefix = '', array $config = []): \Illuminate\Database\Connection
```

Comment:
Please fix the syntax error and revert your changes to the method signature. The signature must match the signature of its parent method.

Reply:
Syntax error fixed, and method signature reverted as requested.